### PR TITLE
Show the heart icon of wish list in product details

### PR DESCRIPTION
**Pull request by ThaynanAndrey**
#### What is the purpose of this pull request?

Show the heart icon of wish list in product details.

#### What problem is this solving?

In product details screen doesn't show the heart icon to add product in wish list.

### How should this be manually tested?

[workspace](https://thaynan--storecomponents.myvtex.com/)

#### Types of changes

* [X] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/AddonWrapper.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import classNames from 'classnames'
+
+const AddonWrapper = () => {
+    const {
+        hints: { mobile },
+    } = useRuntime()
+    const valuesFromContext = useContext(ProductContext)
+    
+    const product= {
+        quantity: 1,
```
Comment (klzns) [inline react/AddonWrapper.js]: Set your indent to 2 spaces, currently it's using 4

File: react/AddonWrapper.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        skuId: path(['selectedItem', 'itemId'], valuesFromContext),
+        productId: path(['product', 'productId'], valuesFromContext)
```
Comment (klzns) [inline react/AddonWrapper.js]: Check the code of:
https://github.com/vtex-apps/product-summary/pull/164/files

You're creating a new `product` object in every render.

File: react/AddonWrapper.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    
+    const isMoreThanOneImage = () => {
```
Comment (klzns) [inline react/AddonWrapper.js]: Use variable instead of function

File: react/ProductDetails.js [hunk 4]
```diff
@@ -389,17 +383,9 @@ class ProductDetails extends Component {
...
-                        id="addon-details-btn"
-                        large={!mobile}
-                        product={{
-                          quantity: 1,
-                          skuId: path(['itemId'], this.selectedItem),
-                          productId: path(['productId'], product),
-                        }}
-                      />
-                    </div>
+                    <ExtensionPoint
```
Comment (klzns) [inline react/ProductDetails.js]: Why this changed?

File: store/interfaces.json [hunk 5]
```diff
@@ -23,6 +23,10 @@
   "product-quantity-selector": {
     "component": "ProductQuantitySelector"
   },
+  "product-addon-wrapper": {
```
Comment (klzns) [inline store/interfaces.json]: Use a more descriptive name 

`product-add-to-list-button` or something like that

File: store/interfaces.json [hunk 6]
```diff
@@ -23,6 +23,10 @@
...
+    "component": "AddonWrapper",
```
Comment (klzns) [inline store/interfaces.json]: Same

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by klzns — APPROVED** [review]

File: react/ProductDetails.js [hunk 7]
```diff
@@ -389,17 +383,9 @@ class ProductDetails extends Component {
                       images={this.getImages()}
                       position={thumbnailPosition}
                     />
-                    <div className={addonDetailsClasses}>
-                      <ExtensionPoint
```
Comment (lucasecdb) [inline react/ProductDetails.js]: shouldn't you still have this? for backwards compatibility

File: react/ProductDetails.js (hunk 4 above)
Comment (klzns) [inline react/ProductDetails.js]: Ops, you should keep the div

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/ProductDetails.js [hunk 8]
```diff
@@ -392,12 +392,6 @@ class ProductDetails extends Component {
                     <div className={addonDetailsClasses}>
                       <ExtensionPoint
                         id="addon-details-btn"
-                        large={!mobile}
```
Comment (lucasecdb) [inline react/ProductDetails.js]: you should also keep the props, else the button won't work

**Review by lucasecdb — APPROVED** [review]

**Review by klzns — APPROVED** [review]

**Pull request merged**